Assignment2/juanbm99-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,59 @@
+{
+	"@context": {
+			"vocab": "http://dispositivos.com#",
+			"xsd": "http://www.w3.org/2001/XMLSchema#",
+			"Class01": "vocab:Class01",
+			"Sensor029": "Class01:Sensor029",
+			"Computer101": "Class01:Computer101",
+			"Measurement8401": "Sensor029:Measurement8401",
+			"User10A": "Computer101:User10A",
```

Comment:
You are defining them as "@type" on your data, thus you don't need to define them here.